conans/test/functional/command/create_test.py

```diff
@@ -200,7 +200,7 @@ def system_requirements(self):
         self.output.info("Running system requirements!!")
 """})
         client.run("create . Pkg/0.1@lasote/testing")
-        self.assertIn("Configuration:[settings]", "".join(str(client.out).splitlines()))
+        self.assertIn("Configuration (host machine):[settings]", "".join(str(client.out).splitlines()))
```

Comment:
I really think that when we are not using cross build profiles everything should work the same, including that message. Maybe I don't have any idea about cross-building or (very likely) don't about gnu triplet jargon and `(host-machines)` means nothing to me.

Reply:
I can remove the `(host machine)` note when both profiles are equal, it is not a big deal (simple change in [manager.py#L56](https://github.com/conan-io/conan/pull/5202/files#diff-9521e09680c5dd9a05e8d76c1bf4d84cR59))